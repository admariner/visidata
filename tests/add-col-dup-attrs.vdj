#!vd -p
{"longname": "open-file", "input": "sample_data/sample.tsv", "keystrokes": "o"}
{"sheet": "sample", "col": "Units", "row": "", "longname": "addcol-new", "input": "integers", "keystrokes": "za", "comment": "append an empty column"}
{"sheet": "sample", "col": "", "row": "", "longname": "select-rows", "input": "", "keystrokes": "gs", "comment": "select all rows"}
{"sheet": "sample", "col": "integers", "row": "", "longname": "setcol-iter", "input": "range(1, 100)", "keystrokes": "gz=", "comment": "set current column for selected rows to the items in result of Python sequence expression"}
{"sheet": "sample", "col": "integers", "row": "", "longname": "type-int", "input": "", "keystrokes": "#", "comment": "set type of current column to int"}
{"sheet": "sample", "col": "integers", "row": "", "longname": "aggregate-col", "input": "mean", "keystrokes": "+", "comment": "Add aggregator to current column"}
{"sheet": "sample", "col": "", "row": "", "longname": "dup-selected", "input": "", "keystrokes": "\"", "comment": "open duplicate sheet with only selected rows"}
{"sheet": "sample_selectedref", "col": "", "row": "", "longname": "columns-sheet", "input": "", "keystrokes": "Shift+C", "comment": "open Columns Sheet: edit column properties for current sheet"}
