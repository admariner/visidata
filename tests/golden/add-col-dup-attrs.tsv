name	width	type	fmtstr	value	aggregators
OrderDate	12			2016-01-06	
Region	9			East	
Rep	10			Jones	
Item	9			Pencil	
Units	7			95	
integers	10	int		1	mean
Unit_Cost	11			1.99	
Total	9			189.05	
